TOMP-API.yaml

```diff
@@ -1784,65 +1866,83 @@ components:
         name:
           description: Human readable name of the place
           type: string
-        stop-reference:
+        stopReference:
           type: array
           items:
-            $ref: '#/components/schemas/stop-reference'
-        station-id:
+            $ref: '#/components/schemas/stopReference'
+        stationId:
           description: reference to /operator/stations
           type: string
-        coordinates:
-          $ref: '#/components/schemas/coordinates'
-        extra-info:
+        coordinate:
+          $ref: '#/components/schemas/coordinate'
+        extraInfo:
           type: array
```

Comment:
The way we do key-value pairs now is structured as:
`
[
  {
    "key1": "value1"
  },
  {
    "key2": "value2"
  },
  {
    "key3": "value3"
  },
]
if the keyValue schema is to be an object, why also the array? It'd be simpler to use a `keyValues` schema like so:
{
  "key1": "value1"
  "key2": "value2"
  "key3": "value3"
}

Reply:
and make it a free format object? Hmmm.... I like the idea. Do it!